src/pages/en/guides/styling.mdx

```diff
@@ -481,6 +481,13 @@ Vue in Astro supports the same methods as `vue-loader` does:
 
 Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-style].
 
+## Markdown Styling
+
+Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts).
+
+You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and [imported stylesheets](#external-styles) to the layout that wraps your page content. You can also add [CSS integrations](#css-integrations) including [Tailwind](/en/guides/integrations-guide/tailwind/).
+
+Additionally, you can install and use [Markdown plugins](/en/guides/markdown-content/#markdown-plugins) such as the [Tailwind/typography plugin](https://tailwindcss.com/docs/typography-plugin) for styling Markdown.
```

Comment:
The tailwind plug-in is a smart thing to call out here but it's not the same kind of plug-in as we document at that link so maybe we only mention tailwind here?

```suggestion
If you are using Tailwind, the [typography plugin](https://tailwindcss.com/docs/typography-plugin) can be useful for styling Markdown.
```